src/app/components/Header/ScriptLink/index.tsx

```diff
@@ -1,19 +1,34 @@
-import React, { useContext } from 'react';
+/** @jsx jsx */
+import { useContext } from 'react';
+import { jsx } from '@emotion/react';
 import { compile } from 'path-to-regexp';
 import clone from 'ramda/src/clone';
 import { useRouteMatch } from 'react-router-dom';
-import ScriptLink from '#psammead/psammead-script-link/src';
 import { UserContext } from '#contexts/UserContext';
 import useToggle from '#hooks/useToggle';
-import { ServiceContext } from '../../../../contexts/ServiceContext';
-import { RequestContext } from '../../../../contexts/RequestContext';
+import { ServiceContext } from '../../../contexts/ServiceContext';
+import { RequestContext } from '../../../contexts/RequestContext';
+import styles from './index.styles';
+
+export type Params = Record<string, string>;
+
+interface UseRouteMatcher {
+  path: string;
+  params: Params;
+}
 
 export const getVariantHref = ({
   path,
   params,
   service,
   variant,
   scriptSwitchId,
+}: {
+  path?: string;
+  params: Params;
+  service: string;
+  variant: string | undefined;
```

Comment:
Should we make variant optional? And also use the correct type?
```suggestion
  variant?: Variants;
```

Reply:
Done from here https://github.com/bbc/simorgh/pull/11867#discussion_r1723080149